tiledb/vfs.py

```diff
@@ -1,11 +1,14 @@
 import io
+import os
 from types import TracebackType
 from typing import List, Optional, Type, Union
 
 import tiledb.cc as lt
 
 from .ctx import Config, Ctx, default_ctx
 
+AnyPath = Union[str, bytes, os.PathLike]
```

Comment:
More of a note to self: We'll want to expand support `pathlib.Path` / path-like objects for other functions like `tiledb.open`, etc. eventually. OK to keep this typing definition in here for now but should probably be moved to a new `_types.py` in the future.

Reply:
In order to make this not look like it is intended to be used by any external users, I renamed this to `_AnyPath`, with no other changes.